documentation/adrs/001-documentation-tool.md

```diff
@@ -0,0 +1,107 @@
+
+### Recommendation
+
+>[!IMPORTANT]
+>The best option at this time is [RSPress].
+>
+> - Active development
+> - Future focused tech stack
+> - Tech stack alignment (with our own)
+
+## Tools
+
+Tool         | Latest    | Releases  | Contributors | Build | Setup | React | Customization | Disqualifier
+:----------- | --------- | --------- | ------------ | ----- | ----- | ----- | ------------: | ------------
+[Docusaurus] | 2 hours   | 148       | 1175         | Yes   |       | Yes   | Moderate      |
+[FumaDocs]   | 2 hours   | 673*      | 43           | Yes   |       | Yes   | Moderate      |
+[RSPress]    | 2 days    | 115       | 97           | Yes   | Easy  | Yes   | Moderate      |
+[Startlight] | 2 days    | 134       | 228          | Yes   |       | Yes   | Moderate      |
+[Vocs]       | 2 months  | 468       | 21           | Yes   |       | Yes   | Moderate      |
+_            |           |           | Disqualified |       |       |       |               | _
+[Docsify]    | 1 day     | 14        | 183          | None  | Easy  | Ext   | Minimal       | Burden of extension
+[Docz]       | 2 years   | 127       | 134          | Yes   |       | Yes   | Moderate      | Age without update
+[JSDoc]      | 2 days    | 12        | 83           | Gen   | Easy  | No    | Extensive     | Burden of extension
+[MkDocs]     | 1 month   | 15        | 244          | Yes   |       |       | Extensive     | Burden of extension
+[Pandoc]     | 1 day     | 140       | 522          | Yes   | Hard  | Ext   | Extensive     | Burden of extension
+[Slate]      | 10 months | 28        | 126          | Yes   | Hard  | No?   | Extensive     | Burden of extension
+[TS-Docs]    | 2 years   | 16        | 1            | Yes   | Hard  | No    | Extensive     | Burden of extension
+[TSDoc]      | 2 weeks   | ???       | 36           | Yes   | Hard  | No?   | Extensive     | Burden of extension
+[TypeDoc]    | 2 days    | 224       | 227          | Yes   |       | No?   | Moderate      | Additional guides support
+
+_Many of the listed tools are quite comparable so choosing one over the other has a narrow margin. However, some are clearly less capable or are less aligned with the goals and desired development practices than others._
+
+### Criteria
+
+- __Build__ - requires a build step to generate the pages
+  - Yes - will be an additional step in CI
+  - Gen - will be an additional step in CI but very basic generation from docblocks
+  - None - will not require a build or generation step
+- __Contributors__ - number of people contributing
+- __Customization__ - the level needed to accomplish our goals
+  - Minimal
+  - Moderate
+  - Extensive
+- __Latest__ - most recent change in github (_at time of collection_)
+- __React__ - renders react components (e.g. from design system)
+  - Yes - native support
+  - Ext - will require an extension/plugin
+- __Releases__ - number of "releases" or production deployments
+- __Setup__ - ease of getting it running: easy, or hard
```

Comment:
One thing I don't see in any of the criteria is search or any kind of discoverability. One of the most difficult parts of docs is finding what you are looking for -- did RSPress have any kind of indexing at all? If we wanted that would we have to build our own plugins?

Reply:
One other requirement that seems reasonable is that whatever we chose, the content shouldn't be "locked" into the platform. Like with Sharepoint it is nearly impossible to move content out of sharepoint and into another context. Although I suppose if most of the writing is in markdown, then that should be open enough.

Reply:
Search is supported "out of the box". 
<img width="385" alt="image" src="https://github.com/user-attachments/assets/9ccd3886-6559-4237-9443-0941c80909cc" />

Great question! And of the "top" contenders search is supported equally I would say.